src/Microsoft.ML.Data/Evaluators/BinaryClassifierEvaluator.cs

```diff
@@ -789,6 +789,34 @@ private void ComputePrCurves()
             }
         }
 
+        /// <summary>
+        /// Generates precision recall curve data points.
+        /// </summary>
+        /// <param name="data">The scored data.</param>
+        /// <param name="label">The name of the label column in <paramref name="data"/>.</param>
+        /// <param name="score">The name of the score column in <paramref name="data"/>.</param>
+        /// <param name="probability">The name of the probability column in <paramref name="data"/>, the calibrated version of <paramref name="score"/>.</param>
+        /// <param name="predictedLabel">The name of the predicted label column in <paramref name="data"/>.</param>
+        /// <returns>The precision recall curve data points.</returns>
+        public IDataView GetPrecisionRecallCurve(IDataView data, string label, string score, string probability, string predictedLabel)
```

Comment:
>string label, string score, string probability, string predictedLabel [](start = 65, length = 69)

Do these column names have default values? If so, let's specify them here as defaults.

Reply:
These are the internal methods that are called by the public `Evaluate` API. The public API does have default values for the column names.

---
In reply to: [267922053](https://github.com/dotnet/machinelearning/pull/3039#discussion_r267922053) [](ancestors = 267922053)

Reply:
as Yael says

---
In reply to: [267926650](https://github.com/dotnet/machinelearning/pull/3039#discussion_r267926650) [](ancestors = 267926650,267922053)